README.md

```diff
@@ -432,11 +432,11 @@ To generalize dateTime, time, date and instant type using expression
 
 ```json
 {
-  "path": "person.birthDate",
+  "path": "Patient.birthDate",
   "method": "generalize",
   "cases":{
-    "$this >= @1990-1-1 and $this <= @2000-1-1": "@1990", 
-     "$this >= @2010-1-1 and $this <= @2020-1-1":"@2010-1-1T00:00:00"
+    "$this >= @1990-01-01 and $this <= @2000-01-01": "@1990", 
+     "$this >= @2010-01-01 and $this <= @2020-01-01":"@2010-01-01"

```

Comment:
Why we remove the timezone info here?

Reply:
Type of birthdate is "date" that shall not has time zone.

Reply:
Make sense. Thanks